tasks/phylogenetic_inference/task_snippy_core.wdl

```diff
@@ -23,21 +26,23 @@ task snippy_core {
    snippy-core \
```

Comment:
add indentation to the lines beneath the snippy-core command for readability; do the same on lines 33-34 too.